native/machine-coding/MadStory_generator/style.css

```diff
@@ -0,0 +1,89 @@
+body {
```

Comment:
import the core styles and also don't modify font family

Reply:
done.